EIPS/eip-tbd.md

```diff
@@ -0,0 +1,76 @@
+---
+eip: 5593
+title: Restrict Web3 Provider Object API Injection
+description: Wallet guidance for restricting web3 provider object API access to secure contexts for improved privacy and security for wallet users.
+author: Yan Zhu (@diracdeltas), Brian R. Bondy (@bbondy), Kyle Den Hartog (@kdenhartog)
+discussions-to:  https://ethereum-magicians.org/t/rfc-limiting-provider-object-injection-to-secure-contexts/10670
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): Interface
+created: 2022-09-05
+requires (*optional): 1193
+---
+
+## Abstract
+
+Historically the web platform has had a notion of “powerful” APIs like [geolocation](https://www.w3.org/TR/geolocation/) and [camera/microphone](https://www.w3.org/TR/mediacapture-streams/), which are subject to additional security restrictions such as those defined by [secure contexts](https://www.w3.org/TR/secure-contexts/). Since the web3 provider object APIs allow dApp websites to request access to sensitive user data and to request use of user funds, new web3 provider object APIs generally should align to the security considerations of other powerful APIs like geolocation.
```

Comment:
@EIPW please suppress warning

I support adding this link. I think this is informative and non-promotional.
FYI: @gcolvin @SamWilsn @Pandapip1

Reply:
Those, again, are not the criterion for determining what is or is not an okay link. The only criterion is "is it going to last longer than the EIPs repository." I would say that links to w3.org are likely okay, but this is not the place to debate that. If you feel like links to w3 are justified, the correct place to do so is in a PR making that change to EIP-1.

Reply:
I'm happy to hold this until we get that resolved and will contribute to the discussion as well based on my experience as an editor in W3C.